doc/changes.md

```diff
@@ -1,5 +1,111 @@
 # Changelog
 
+
+## Release v0.8.0 (2022/12/28)
+
+[![Digital Object Identifier for PyGMT v0.8.0](https://zenodo.org/badge/DOI/10.5281/zenodo.7481934.svg)](https://doi.org/10.5281/zenodo.7481934)
+
+### Highlights
+
+* 🎉 **Eighth minor release of PyGMT** 🎉
+* Wrapped ternary
+* Add a intro tutorial for creating contour maps
+* Easy access to various remote datasets
+
+### New Features
+
+* Add load_earth_vertical_gravity_gradient function for Earth vertical gravity gradient dataset ([#2240](https://github.com/GenericMappingTools/pygmt/pull/2240))
+* Add load_earth_free_air_anomaly function for Earth free-air anomaly dataset ([#2238](https://github.com/GenericMappingTools/pygmt/pull/2238))
+* Add load_earth_geoid function for Earth Geoid dataset ([#2236](https://github.com/GenericMappingTools/pygmt/pull/2236))
+* Add load_earth_magnetic_anomaly function for Earth magnetic anomaly dataset ([#2196](https://github.com/GenericMappingTools/pygmt/pull/2196), [#2239](https://github.com/GenericMappingTools/pygmt/pull/2239))
+* load_earth_relief: Add the support of data sources 'gebco' , 'gebcosi', and 'synbath' ([#1818](https://github.com/GenericMappingTools/pygmt/pull/1818), [#2162](https://github.com/GenericMappingTools/pygmt/pull/2162), [#2192](https://github.com/GenericMappingTools/pygmt/pull/2192))
+* Wrap ternary ([#1431](https://github.com/GenericMappingTools/pygmt/pull/1431))
+
+### Enhancements
+
+* Add ternary sample dataset ([#2211](https://github.com/GenericMappingTools/pygmt/pull/2211))
+* pygmt.config: Support tab auto-completion for all GMT defaults ([#2213](https://github.com/GenericMappingTools/pygmt/pull/2213))
```

Comment:
What about highlighting the auto-completion of GMT defaults?

Reply:
I have added it to the "Highlights" section in commit https://github.com/GenericMappingTools/pygmt/pull/2272/commits/3acad82423f46e6dd85a727553e4c44df938eaf0.
I personally feel we should not have more than three or four entries in the Highlights section addional to "Eighth minor release of PyGMT".

-------------
Does point 6 of the section [Updating the Changelog](https://www.pygmt.org/dev/maintenance.html#updating-the-changelog) in the Maintainers Guide mean, that we remove the Highlight PRs from the sections below?
> Move a few important items from the main sections to the highlights section.